darwin/importer/importer.py

```diff
@@ -128,7 +128,7 @@ def get_remote_files(dataset: "RemoteDataset", filenames: List[str], chunk_size:
         for remote_file in dataset.fetch_remote_files(
             {"types": "image,playback_video,video_frame", "filenames": chunk}
         ):
-            remote_files[remote_file.full_path] = remote_file.id
+            remote_files[remote_file.full_path.strip("/")] = remote_file.id
```

Comment:
Isn't this a breaking change? This code is working on `master` and it feels like there is no way that both versions are fine.